[Audio] Error when changing Java path

# Command bugs

<!-- 
Did you find a bug with a command? Fill out the following:
-->

#### Command name

`llset java`

#### What cog is this command from?

Audio

#### What were you expecting to happen?

To set a custom java path

#### What actually happened?

No response in Discord, traceback in the console

#### How can we reproduce this issue?

`llset java /usr/bin/java` or any other executable path (I get the expected error message in Discord if provided path is not a valid executable)

```
[2020-11-18 18:44:58] [ERROR] red: TypeError
Traceback (most recent call last):
File "/home/red/envs/eevee/lib/python3.8/site-packages/discord/ext/commands/core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
File "/home/red/envs/eevee/lib/python3.8/site-packages/redbot/cogs/audio/core/commands/llset.py", line 58, in command_llsetup_java
    await self.config.java_exc_path.set(exc_absolute)
File "/home/red/envs/eevee/lib/python3.8/site-packages/redbot/core/config.py", line 257, in set
    await self.driver.set(self.identifier_data, value=value)
File "/home/red/envs/eevee/lib/python3.8/site-packages/redbot/core/drivers/json.py", line 146, in set
    value_copy = json.loads(json.dumps(value))
File "/usr/lib/python3.8/json/init.py", line 231, in dumps
    return _default_encoder.encode(obj)
File "/usr/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
File "/usr/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
File "/usr/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.class.name} '
TypeError: Object of type PosixPath is not JSON serializable
```

--- a/redbot/cogs/audio/core/commands/llset.py
+++ b/redbot/cogs/audio/core/commands/llset.py
@@ -55,7 +55,7 @@ class LavalinkSetupCommands(MixinMeta, metaclass=CompositeMetaClass):
                         java_path=exc_absolute
                     ),
                 )
-            await self.config.java_exc_path.set(exc_absolute)
+            await self.config.java_exc_path.set(str(exc_absolute))
             await self.send_embed_msg(
                 ctx,
                 title=_("Java Executable Changed"),

